src/ddmd/root/array.d

```diff
@@ -199,12 +199,12 @@ public:
 struct BitArray
 {
 nothrow:
-    size_t length() const pure
+    @property size_t length() const pure
```

Comment:
I'd like to leave the `@property` here because it is one.
Any other blockers @andralex